docs/changes.rst

```diff
@@ -1,6 +1,75 @@
 Changelog
 =========
 
+Version 1.1.0 (not released yet)
+--------------------------------
+
+New features:
+
+- borg recreate: re-create existing archives, #787 #686 #630 #70, also see
+  #757, #770.
+
+  - selectively remove files/dirs from old archives
+  - re-compress data
+  - re-chunkify data, e.g. to have upgraded Attic / Borg 0.xx archives
+    deduplicate with Borg 1.x archives or to experiment with chunker-params.
+- create: visit files in inode order (better speed, esp. for large directories
+  and rotating disks)
+- borg diff: show differences between archives
+- borg list improved:
+
+  - removed {formatkeys}, see "borg list --help"
+  - --list-format is deprecated, use --format instead
+  - now supports the usual [PATH [PATHS…]] syntax and excludes
+  - new keys: csize, num_chunks, unique_chunks, NUL
+  - supports guaranteed_available hashlib hashes
+    (to avoid varying functionality depending on environment)
+- prune:
+
+  - to better visualize the "thinning out", we now list all archives in
+    reverse time order. rephrase and reorder help text.
+  - implement --keep-last N via --keep-secondly N, also --keep-minutely.
+    assuming that there is not more than 1 backup archive made in 1s,
```

Comment:
On many fonts `1s` (one-ess) looks like `ls` (ell-ess), which confused me for a second. Changing it to `1 second` avoids the ambiguity.